### ISSUE TITLE
NPM's latest version has broken Parser.js. Does not handle monochrome png's

### ISSUE THREAD

**Issue by ghost** [gagern/jspngopt#1]
A png was created using `pngquant 2 foo.png`

the lines in Parser.js responsible are 36 & 37
```
  // if ([8].indexOf(hdr.bitDepth) === -1)
  //   throw Error("Unsupported bit depth: " + hdr.bitDepth);
```

I see the Parser.js here on github has been fixed.

**Comment by gagern** [comment]
Yes, I should roll a new release. I intend to do so once I got your #4 in.

**Comment by gagern** [comment]
I just released [version 0.2.0](https://github.com/gagern/jspngopt/releases/tag/0.2.0).
